src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVVCFWriter.java

```diff
@@ -33,15 +34,27 @@
      * sequence dictionaries in a scrambled order, see https://github.com/broadinstitute/gatk/issues/2037.
      */
     static void writeVCF(final PipelineOptions pipelineOptions, final String outputPath, String vcfFileName,
-                         final String fastaReference, final JavaRDD<VariantContext> variantContexts) {
+                         final String fastaReference, final JavaRDD<VariantContext> variantContexts,
+                         final Logger logger) {
 
         final SAMSequenceDictionary referenceSequenceDictionary = new ReferenceMultiSource(pipelineOptions, fastaReference, ReferenceWindowFunctions.IDENTITY_FUNCTION).getReferenceSequenceDictionary(null);
 
         final List<VariantContext> sortedVariantsList = sortVariantsByCoordinate(variantContexts.collect(), referenceSequenceDictionary);
 
+        logNumOfVarByTypes(sortedVariantsList, logger);
+
         writeVariants(pipelineOptions, outputPath, vcfFileName, sortedVariantsList, referenceSequenceDictionary);
     }
 
+    private static void logNumOfVarByTypes(final List<VariantContext> sortedVariantsList, final Logger logger) {
+
+        logger.info("Discovered " + sortedVariantsList.size() + " variants.");
+
+        sortedVariantsList.stream()
+                .collect(Collectors.groupingBy(vc -> (String)vc.getAttribute(GATKSVVCFHeaderLines.SVTYPE)))
+                .entrySet().forEach(pair -> logger.info(pair.getValue().size() + " "+ pair.getKey() + "'s"));
```

Comment:
I'm not sure, but it might be more efficient to use a counting collector here. I think it would look like this (not tested):

```
sortedVariantsList.stream()
   .collect(Collectors.groupingBy(vc -> (String)vc.getAttribute(GATKSVVCFHeaderLines.SVTYPE)), Collectors.counting())
  .entrySet().forEach(pair -> logger.info(pair.getKey() + ": " + pair.getValue"));

```